src/routes/dashboard/settings/+page.server.ts

```diff
@@ -47,4 +60,66 @@ export const actions: Actions = {
             },
         };
     },
+    changePassword: async ({ cookies, request }) => {
+        const formData = await request.formData();
+        const token = cookies.get('token');
+        if (!token) {
+            return redirect(301, '/login');
+        }
+        const newPassword = formData.get('newPassword');
+        const confirmPassword = formData.get('confirmPassword');
+
+        if (
+            newPassword === '' ||
+            confirmPassword === '' ||
```

Comment:
Why not simply `if (!newPassword || !confirmPassword)`

Reply:
well that also works 🫠